ledger/acctupdates.go

```diff
@@ -2384,8 +2384,16 @@ func (au *accountUpdates) latest() basics.Round {
 	return au.dbRound + basics.Round(len(au.deltas))
 }
 
+func (au *accountUpdates) generateCatchpointAsync(committedRound basics.Round, label string, committedRoundDigest crypto.Digest, updatingBalancesDuration time.Duration) {
+	done := make(chan struct{})
```

Comment:
make this into a buffered channel of size of 1, and add a `defer close(async)` in `generateCatchpoint` to ensure that in any case the method exits, you don't get stuck leaving zombies.